changelog/README.md

```diff
@@ -0,0 +1,59 @@
+# Multi-file running changelog
+
+To avoid merge conflicts in the CHANGELOG.md file we accepted the multi-file running changelog strategy.
+
+To follow this strategy you should create a `changelog/unreleased/${PR_NUMBER}.json` file for every PR like:
```

Comment:
we usually add many lines during bump of NN and common dependencies. Does it mean that we will have to add as many files as many line we want to add during update of NN?

Reply:
yeah, if we move features/bugfixes type to the path we can add files with additional info like this in the unreleased directory and structure will be like:
```
changelog
--unreleased
----features
------feature_1.md
----bugfixes
------bugfix_1.md
----other_change_1.md
----other_change_2.md
```